packages/flame/lib/src/game/game_widget/game_widget.dart

```diff
@@ -288,7 +288,7 @@ class _GameWidgetState<T extends Game> extends State<GameWidget<T>> {
     if (game is KeyboardEvents) {
       return game.onKeyEvent(event, RawKeyboard.instance.keysPressed);
     }
-    return KeyEventResult.handled;
+    return KeyEventResult.ignored;
```

Comment:
I think there was a problem with having this as `ignored`, but I don't remember what it was... @renancaraujo do you remember?

Reply:
This is because of the very annoying system error that happens if a key event is ignored by the app. If the game has focus, the system should always assume that all key events are handled (unless specified by the game). If we want other widgets (outside gamewidget/overlays) to handle key events, they should handle focus.

As of the bug that this pr want to solve, I dont think this is the right way (by the reason stated above). We should rather figure out a way to only handle things in here if the gamewidgets focusnode has the primary focus.